app/Resources/translations/default/EmailsBody.xlf

```diff
@@ -1,1179 +1,2187 @@
+        <target>We wish to thank you for the trust you have placed in us and want to give you a discount of [bold]{amount}%[/bold] valid on your next order! This offer is valid for [bold]{days}[/bold] days, so do not waste a moment!</target>
+        <note>Line: 31</note>
       </trans-unit>
     </body>
   </file>
-  <file original="modules/ps_emailgenerator/templates/modules/ps_emailalerts/customer_qty.php" source-language="en-US" target-language="en" datatype="plaintext">
+  <file original="mails/themes/classic/modules/ps_emailalerts/customer_qty.html.twig" source-language="en-US" target-language="en" datatype="plaintext">
     <body>
-      <trans-unit id="9b096ae55ce4d2bb889392144c104069">
-        <source>{product} is now available.</source>
-        <target>{product} is now available.</target>
-        <note>Context:
-File: modules/ps_emailgenerator/templates/modules/ps_emailalerts/customer_qty.php:21</note>
-      </trans-unit>
       <trans-unit id="5b736bebe50e10240ff58d486494dc7b">
         <source>This item is once again in-stock.</source>
         <target>This item is once again in-stock.</target>
-        <note>Context:
-File: modules/ps_emailgenerator/templates/modules/ps_emailalerts/customer_qty.php:24</note>
+        <note>Line: 29</note>
       </trans-unit>
       <trans-unit id="16ce091d5fafd5b92941221eab85f3af">
         <source>You can access the product page by clicking on the link:</source>
         <target>You can access the product page by clicking on the link:</target>
-        <note>Context:
-File: modules/ps_emailgenerator/templates/modules/ps_emailalerts/customer_qty.php:25</note>
+        <note>Line: 30</note>
       </trans-unit>
       <trans-unit id="d5c8de069674d35946f2e88642635485">
         <source>You can order it right now from our online shop.</source>
         <target>You can order it right now from our online shop.</target>
-        <note>Context:
-File: modules/ps_emailgenerator/templates/modules/ps_emailalerts/customer_qty.php:26</note>
+        <note>Line: 31</note>
       </trans-unit>
     </body>
   </file>
-  <file original="modules/ps_emailgenerator/templates/modules/ps_emailalerts/new_order.php" source-language="en-US" target-language="en" datatype="plaintext">
+  <file original="mails/themes/classic/modules/ps_emailalerts/new_order.html.twig" source-language="en-US" target-language="en" datatype="plaintext">
     <body>
-      <trans-unit id="5c1cc11c191c9a0f32d16439618ed3bb">
-        <source>Customer message:</source>
-        <target>Customer message:</target>
-        <note>Context:
-File: modules/ps_emailgenerator/templates/modules/ps_emailalerts/new_order.php:312</note>
-      </trans-unit>
-      <trans-unit id="f5d74ea75357b5e139854c14f8e24fe3">
-        <source>Order details</source>
-        <ta
```

Comment:
@eternoendless I'm a bit worried about this key because it was extracted with `Advanced Parameters > Logs` whereas in the template I used `&gt;`
```php
{{ 'You can check for it in the [bold]Advanced Parameters &gt; Logs[/bold] section of your back office.'|trans({'[bold]': '<span><strong>', '[/bold]': '</strong></span>'}, 'Emails.Body', locale)|raw }}
```
I'm worried the key won't match

Reply:
I think it's just an encoding thing. You can test the translation by copying this file to the appropriate locale, then writing a translation in "target". You may need to reset your cache.

The `trans-unit id` is just an md5 of the source string, it looks like it matches the source string as it's originally written:

```
$ md5 -s "You can check for it in the [bold]Advanced Parameters &gt; Logs[/bold] section of your back office."
MD5 ("You can check for it in the [bold]Advanced Parameters &gt; Logs[/bold] section of your back office.") = 94cfcd125e4af439bb58f46fbbb48dfa

```